exercises/en/exc_03_09a.py

```diff
@@ -0,0 +1,17 @@
+import altair as alt
+import pandas as pd
+
+
+penguins = pd.read_csv('data/penguins.csv')
+
+mass_density_plot = alt.Chart(___).transform_density(
+    ___,
+    ___,
+    ___,
+    ___
```

Comment:
I think we need some more hints here, as this is the learners first time writing this code.